src/python/qgspythonutilsimpl.cpp

```diff
@@ -145,23 +145,49 @@ bool QgsPythonUtilsImpl::checkSystemImports()
 
 void QgsPythonUtilsImpl::init()
 {
+
+
+#ifdef Q_OS_WIN
+  // Fix #45646 On Windows, sys.executable is not set correctly
+  // it points to the qgis executable instead of python3.exe
+  // See related discussion https://github.com/python/cpython/issues/78906
+  std::wstring pythonPath = QDir::toNativeSeparators( QString( "%1/python3.exe" ).arg( QCoreApplication::applicationDirPath() ) ).toStdWString();
```

Comment:
wondering if this isn't too specific to OSGeo4W layout ?
@SrNetoChan is python3.exe located in the same directory as the QGIS executable on Conda-Forge builds ?
Otherwise if the above filename wouldn't exit, I guess https://learn.microsoft.com/en-us/windows/win32/api/processenv/nf-processenv-searchpathw could potentially be used to locate it.

Reply:
Ping @SrNetoChan

Reply:
@rouault @nyalldawson Sorry, I was missing the github notifications on my email. Yes, in conda builds the python executable is in the same directory as the qgis executable,  but it's called `python.exe`, not `python3.exe`

```
(base) C:\Users\aneto>mamba activate qgis

(qgis) C:\Users\aneto>where qgis
C:\Users\aneto\mambaforge\envs\qgis\Library\bin\qgis.exe

(qgis) C:\Users\aneto>where python
C:\Users\aneto\mambaforge\envs\qgis\python.exe
C:\Users\aneto\AppData\Local\Microsoft\WindowsApps\python.exe
```